cob_bringup/env.sh

```diff
@@ -13,4 +13,6 @@ else
     source /opt/ros/indigo/setup.bash
 fi
 
+export ROS_IP=`hostname -I | awk '{print $1}'`
```

Comment:
In principle should be ok, was already tested on the robot? also sourcing the env.sh of other user to execute the robot.launch?